### PR TITLE
Tweak opencl launch params v2

### PR DESCRIPTION
**Pull request by Tongdongq**
Same as https://github.com/OPM/opm-simulators/pull/3953, but without the compile error.
Also reduced the length of some more lines.

### CODE REVIEW

**Comment by bska** [comment]
jenkins build this please

File: opm/simulators/linalg/bda/opencl/openclKernels.hpp [hunk 1]
```diff
@@ -64,6 +64,7 @@ class OpenclKernels
     static cl::CommandQueue *queue;
     static std::vector<double> tmp;     // used as tmp CPU buffer for dot() and norm()
     static bool initialized;
+    static size_t preferred_workgroup_size_multiple; // stores CL_KERNEL_PREFERRED_WORK_GROUP_SIZE_MULTIPLE
```
Comment (bska) [inline opm/simulators/linalg/bda/opencl/openclKernels.hpp:67]: You should include `<stddef.h>` if you have data members of type `size_t`.  Alternatively, you could/should make the data member into a `std::size_t` and include `<cstddef>` instead.

Comment (Tongdongq) [reply opm/simulators/linalg/bda/opencl/openclKernels.hpp]: 18d9f8c, uses `<cstddef>`.

**Comment by bska** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks for the updates.  This looks good to me now and I'll merge into master.

**Pull request merged**